public/app/features/dashboard-scene/utils/clone.ts

```diff
@@ -0,0 +1,73 @@
+const CLONE_KEY = '-clone-';
+const CLONE_SEPARATOR = '|';
```

Comment:
can we change this to `/` to matter signify the hierarchy

Reply:
Done!